cms/templates/js/course-outline.underscore

```diff
@@ -4,6 +4,32 @@ var visibilityState = xblockInfo.get('visibility_state');
 var published = xblockInfo.get('published');
 var prereq = xblockInfo.get('prereq');
 var hasPartitionGroups = xblockInfo.get('has_partition_group_components');
+var userPartitions = xblockInfo.get('user_partitions');
+var groupAccess = xblockInfo.get('group_access');
+var allowedGroups = [];
+var groupIdToName = {};
+
+for (var schemeId in groupAccess) {
+    var schemeAllowedGroups = groupAccess[schemeId];
+
+    schemeAllowedGroups.forEach(function (groupId) {
+        var fullId = schemeId + groupId;
```

Comment:
Isn't this concatenating together 2 integers? If so, that seems like a bad idea because '3' + '10' == '31' + '0', for example.